filament/src/components/RenderableManager.cpp

```diff
@@ -596,14 +596,22 @@ static void updateMorphWeights(FEngine& engine, backend::Handle<backend::HwBuffe
     driver.updateBufferObject(handle, { out, size }, 0);
 }
 
-void FRenderableManager::setMorphWeights(Instance instance, float const* weights, size_t count) noexcept {
+void FRenderableManager::setMorphTargetCount(Instance instance, size_t count) noexcept {
     if (instance) {
+        ASSERT_PRECONDITION(count < CONFIG_MAX_MORPH_TARGET_COUNT,
```

Comment:
when using `ASSERT_*` the method cannot be `noexcept`, because these will throw if exceptions are enabled.

Reply:
Thanks it's updated.